### PR TITLE
Let system handle ethtool location

### LINKED ISSUE #508: networking/info_device(number)_(interface) can't find ethtool in /sbin/ethtool on Desktop
### Bug Description

Checkbox22 snap latest/stable version 2.6

The test: network/info_device(number)_(interface) attempts to get some information about a given network interface (e.g. enp1s0) using ethtool which is provided by Checkbox in /snap/checkbox22/current/sbin/ethtool

On Desktop this test fails because it claims "ethtool is not installed."
On Server this test passes but it says "ethtool returned an error." 

This test uses the script: https://github.com/canonical/checkbox/blob/main/providers/base/bin/network_device_info.py

This script calls /sbin/ethtool with the interface to be probed.

This occurs in this block of code:
```52    cmd = ['/sbin/ethtool', interface]
53        try:
54            output = check_output(cmd, stderr=STDOUT, universal_newlines=True)
55        except CalledProcessError:
56            return "ethtool returned error"
57        except FileNotFoundError:
58            return "ethtool not installed"
59        if not output:
60            return "ethtool returned no output" ```

When checkbox calls this script, the FileNotFoundError exception is caught which results in this output in the submission file:

Category: NETWORK
Interface: enp1s0
Product: Ethernet Controller I225-LM (Ethernet Network Adapter I225-T1)
Vendor: Intel Corporation
Driver: igc
Driver Version: 
Path: /devices/pci0000:00/0000:00:1c.0/0000:01:00.0
Id: [8086:15f2]
Subsystem Id: [8086:0001]
Mac: 40:a6:b7:6a:19:6b
Carrier Status: Connected
Ipv4: 10.102.88.104
Ipv6: fe80::2c00:ed07:e484:9779/64
Speed: 1000
Supported Modes: ethtool not installed
Advertised Modes: ethtool not installed
Partner Modes: ethtool not installed

But when I manually call this inside a checkbox shell, it works:
u@u:~$ checkbox-iiotg-classic.shell 
checkbox-iiotg-classic runtime shell, type 'exit' to quit the session
u@u:~$ sudo ethtool enp1s0
Settings for enp1s0:
	Supported ports: [  ]
	Supported link modes:   10baseT/Half 10baseT/Full
	                        100baseT/Half 100baseT/Full
	                        1000baseT/Full
	                        2500baseT/Full
	Supported pause frame use: Symmetric
	Supports auto-negotiation: Yes
	Supported FEC modes: Not reported
	Advertised link modes:  10baseT/Half 10baseT/Full
	                        100baseT/Half 100baseT/Full
	                        1000baseT/Full
	                        2500baseT/Full
	Advertised pause frame use: Symmetric
	Advertised auto-negotiation: Yes
	Advertised FEC modes: Not reported
	Speed: 1000Mb/s
	Duplex: Full
	Auto-negotiation: on
	Port: Twisted Pair
	PHYAD: 0
	Transceiver: internal
	MDI-X: off (auto)
	Supports Wake-on: pumbg
	Wake-on: g
        Current message level: 0x00000007 (7)
                               drv probe link
	Link detected: yes

### To Reproduce

checkbox-iiotg-class.test-runner
choose Desktop Manual
choose non-device specific networking test

Tests should pass.
Tests fail with "ethtool not installed."

### Environment

22.04 Jammy IOTG Desktop with IOTG RT-GA kernel
checkbox22 v2.6
checkbox-iiotg-classic 0.1dev-jammy  (latest/edge)


### Relevant log output

```shell
I/O log:
Category: NETWORK
Interface: enp1s0
Product: Ethernet Controller I225-LM (Ethernet Network Adapter I225-T1)
Vendor: Intel Corporation
Driver: igc
Driver Version: 
Path: /devices/pci0000:00/0000:00:1c.0/0000:01:00.0
Id: [8086:15f2]
Subsystem Id: [8086:0001]
Mac: 40:a6:b7:6a:19:6b
Carrier Status: Connected
Ipv4: 10.102.88.104
Ipv6: fe80::2c00:ed07:e484:9779/64
Speed: 1000
Supported Modes: ethtool not installed
Advertised Modes: ethtool not installed
Partner Modes: ethtool not installed
```

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by pieqq**
## Description

Trying to call `/sbin/ethtool` causes our job `networking/info_device(number)_(interface)` to fail loading additional information when Checkbox is installed as a snap, because in this case the location of ethtool is `/snap/checkbox22/current/sbin/ethtool`.

Calling ethtool directly to let the system handle the tool location.

## Resolved issues

Fix #508

## Tests

Tested the change locally in a venv:

```
./network_device_info.py info NETWORK --interface enp0s31f6
Category: NETWORK
Interface: enp0s31f6
Product: Ethernet Connection (16) I219-LM
Vendor: Intel Corporation
Driver: e1000e
Driver Version: 
Path: /devices/pci0000:00/0000:00:1f.6
Id: [8086:1a1e]
Subsystem Id: [1028:0b08]
Mac: 00:be:43:42:6a:36
Carrier Status: Connected
Ipv4: 10.102.88.116
Ipv6: fe80::cd42:fe94:9a6d:7719/64
Speed: 1000
Supported Modes: 
	10baseT/Half 10baseT/Full
	100baseT/Half 100baseT/Full
	1000baseT/Full
Advertised Modes: 
	10baseT/Half 10baseT/Full
	100baseT/Half 100baseT/Full
	1000baseT/Full
Partner Modes: 
```

In the checkbox22 snap, ethtool resides in `/snap/checkbox22/current/sbin/ethtool`. Calling it directly rather that with `/sbin/ethtool` will fix the issue.

### CODE REVIEW

**Review by zongminl — APPROVED** [review]
LGTM, thank you for this fix.

**Review by Hook25 — APPROVED** [review]
duplicating +1 as @zongminl can not approve this (he does not have write access on this repo it seems)

**Pull request merged**
